packages/core/admin/index.js

```diff
@@ -92,17 +92,29 @@ async function build({ plugins, dir, env, options, optimize }) {
 async function createPluginsJs(plugins, dest) {
   const pluginsArray = plugins.map(({ pathToPlugin, name }) => {
     const shortName = _.camelCase(name);
+
+    /**
+     * path.join, on windows, it uses backslashes to resolve path.
+     * The problem is that JavaScript does not resolve the path as windows.
+     * In JavaScript, we need to rely on "/" and not "\".
+     * This is the reason why '..\\..\\..\\node_modules\\@strapi\\plugin-content-type-builder/strapi-admin.js' was not working in javascript.
+     * The regexp at line 105 aims to replace the windows backslashes by standard slash so that javascript can deal with them.
+     */
+    const realPath = path
+      .join(path.relative(path.resolve(dest, 'admin/src'), pathToPlugin), 'strapi-admin.js')
+      .replace(/\\/g, '/');
+
     return {
       name,
-      pathToPlugin: path.relative(path.resolve(dest, 'admin/src'), pathToPlugin),
+      pathToPlugin: realPath,
       shortName,
     };
   });
 
   const content = `
 ${pluginsArray
   .map(({ pathToPlugin, shortName }) => {
-    const req = `'${pathToPlugin}/strapi-admin.js'`;
```

Comment:
Does it work if you only do :
```js
const req = path.join(pathToPlugin, 'strapi-admin.js')
```
?

Reply:
it should if the replace all call comes after this line :)

Reply:
I'm not sure I understood. What do you mean ?

Reply:
Sorry, I'll try to be clearer. It won't work since path.join will resolve the path with backslash on windows. Like `somewhere\strapi-admin.js`

Reply:
I think Node handles both. The problem there seems to me that we had a mix of both `pathToPlugin` (windows) + `/strapi-admin.js` (linux)

Reply:
I'll try your suggestion tomorrow morning and check if it works (without mixing), thanks for the suggestion :D

Reply:
I'll just try with the following: 

```javascript
  const pluginsArray = plugins.map(({ pathToPlugin, name }) => {
    const shortName = _.camelCase(name);
    const combinedPath = path.relative(path.resolve(dest, 'admin', 'src'), pathToPlugin);

    return {
      name,
      pathToPlugin: combinedPath,
      shortName,
    };
  });

  const content = `
${pluginsArray
  .map(({ pathToPlugin, shortName }) => {
    const req = `'${path.join(pathToPlugin, 'strapi-admin.js')}'`;

    return `import ${shortName} from ${req};`;
  })
  .join('\n')}


const plugins = {
${[...pluginsArray]
  .map(({ name, shortName }) => {
    return `  '${name}': ${shortName},`;
  })
  .join('\n')}
};

export default plugins;
`;
```

but I got the same error as before the replace all with the following path:

```js
import contentTypeBuilder from '..\..\..\node_modules\@strapi\plugin-content-type-builder\strapi-admin.js';
```